### PR TITLE
Kill jdwp

### PR DESCRIPTION
**Pull request by chrisforbes**

### CODE REVIEW

**Comment by chrisforbes** [comment]
This builds on top of the previous change to remove the GLES API. Only the final commit is interesting.

File: gapii/vulkan/vk_graphics_spy/cc/layer.cpp [hunk 1]
```diff
@@ -36,21 +36,9 @@ typedef void* (*eglGetProcAddress)(const char* procname);
```
Comment (hevrard) [inline gapii/vulkan/vk_graphics_spy/cc/layer.cpp]: on L28, remove: `typedef void* (*eglGetProcAddress)(const char* procname);`

Comment (chrisforbes) [reply gapii/vulkan/vk_graphics_spy/cc/layer.cpp]: Done.

**Comment by chrisforbes** [comment]
[trivial rebase]

**Pull request merged**